serialisation/src/test/java/org/ehrbase/serialisation/flatencoding/StructuredJsonTest.java

```diff
@@ -21,72 +21,62 @@
 import org.ehrbase.test_data.composition.CompositionTestDataStructuredJson;
 import org.ehrbase.test_data.operationaltemplate.OperationalTemplateTestData;
-import org.ehrbase.validation.Validator;
 import org.junit.jupiter.api.Test;
 import org.skyscreamer.jsonassert.JSONAssert;
 import org.skyscreamer.jsonassert.JSONCompareMode;
 
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-
 class StructuredJsonTest {
-    private static final TestDataTemplateProvider templateProvider = new TestDataTemplateProvider();
-
-    @Test
-   void testRoundTrip() throws IOException {
-
-        CompositionTestDataStructuredJson testData = CompositionTestDataStructuredJson.MULTI_LIST;
-        String templateId = OperationalTemplateTestData.MULTI_LIST.getTemplateId();
-
-
-        test(testData, templateId);
-    }
 
-    @Test
-    void testRoundTripCorona() throws IOException {
+  private static final TestDataTemplateProvider templateProvider = new TestDataTemplateProvider();
 
-        CompositionTestDataStructuredJson testData = CompositionTestDataStructuredJson.CORONA;
-        String templateId = OperationalTemplateTestData.CORONA_ANAMNESE.getTemplateId();
+  @Test
+  void testRoundTrip() throws IOException {
+    CompositionTestDataStructuredJson testData = CompositionTestDataStructuredJson.MULTI_LIST;
+    String templateId = OperationalTemplateTestData.MULTI_LIST.getTemplateId();
 
+    test(testData, templateId);
+  }
 
-        test(testData, templateId);
-    }
+  @Test
+  void testRoundTripCorona() throws IOException {
+    CompositionTestDataStructuredJson testData = CompositionTestDataStructuredJson.CORONA;
+    String templateId = OperationalTemplateTestData.CORONA_ANAMNESE.getTemplateId();
 
-    private void test(CompositionTestDataStructuredJson testData, String templateId) throws IOException {
-        RMDataFormat cut =
-                new FlatJasonProvider(templateProvider).buildFlatJson(FlatFormat.STRUCTURED, templateId);
+    test(testData, templateId);
+  }
 
-        String flat = IOUtils.toString(testData.getStream(), StandardCharsets.UTF_8);
-        Composition unmarshal = cut.unmarshal(flat);
+  private void test(CompositionTestDataStructuredJson testData, String templateId)
+      throws IOException {
+    RMDataFormat cut =
+        new FlatJasonProvider(templateProvider).buildFlatJson(FlatFormat.STRUCTURED, templateId);
 
-        SoftAssertions softAssertions = new SoftAssertions();
+    String flat = IOUtils.toString(testData.getStream(), StandardCharsets.UTF_8);
+    Composition unmarshal = cut.unmarshal(flat);
 
-        softAssertions.assertThat(unmarshal).isNotNull();
+    SoftAssertions softAssertions = new SoftAssertions();
 
-        try {
-            new Validator(templateProvider.find(templateId).get()).check(unmarshal);
-        } catch (Exception e) {
-            softAssertions.fail(e.getMessage());
-        }
+    softAssertions.assertThat(unmarshal).isNotNull();
 
```

Comment:
Please add the new compostionValidate where ever you removed the old one.

Reply:
Tests added in `StructuredJsonValidationTest`